ui-v2/app/styles/components/healthcheck-output.scss

```diff
@@ -0,0 +1,35 @@
+@import './healthcheck-output/index';
+@import './icons/index';
+.healthcheck-output {
+  @extend %healthcheck-output;
+}
+%healthcheck-output.passing {
+  @extend %with-passing;
+}
+%healthcheck-output.warning {
+  @extend %with-warning;
+}
+%healthcheck-output.critical {
+  @extend %with-critical;
+}
+@media #{$--lt-spacious-healthcheck-output} {
+  .healthcheck-output button.copy-btn {
+    margin-top: -11px;
+    margin-right: -18px;
+    padding: 0;
+    width: 20px;
+    visibility: hidden;
+  }
+  %healthcheck-output {
+    padding-left: 30px;
+    padding-top: 10px;
+    padding-bottom: 15px;
+    padding-right: 13px;
+  }
+  %healthcheck-output::before {
+    width: 15px !important;
+    height: 15px !important;
+    left: 9px;
+    top: 13px !important;
+  }
+}
```

Comment:
This addition is a consequence of moving this `healthcheck-output` into a component, therefore I've given it is own corresponding CSS component - the code here already existed.